PWGGA/GammaConvBase/AliCaloPhotonCuts.cxx

```diff
@@ -2431,6 +2440,39 @@ Bool_t AliCaloPhotonCuts::ClusterQualityCuts(AliVCluster* cluster, AliVEvent *ev
     if ( classification == 3)
       fHistClusterTMEffiInput->Fill(cluster->E(), 5., weight); // Ga cl sub ch match
 
+      // check if cluster has an associated track
+      Bool_t isValidatedTrack = kFALSE;
+      std::vector<Int_t> vecMatchedTracks = GetVectorMatchedTracksToCluster(event, cluster);
+      if(vecMatchedTracks.size() > 0){
+        for(UInt_t itrack = 0; itrack < vecMatchedTracks.size(); ++itrack ){
+          AliVTrack* currTrack  = dynamic_cast<AliVTrack*>(event->GetTrack(vecMatchedTracks.at(0)));
```

Comment:
shouldn't there be 'itrack' instead of '0' in vecMatchedTracks.at(...) ?

Reply:
Yes that is true! Thanks for catching that mistake! I corrected it